docs/getting-started.md

```diff
@@ -24,22 +24,22 @@ npx react-native init <projectName> --template react-native@^0.63.2
 
 Once your project has been initialized, React Native will have created a new sub directory where all your generated files live.
 
-```
+```cmd

```

Comment:
nice! :)

We should probably change the `<projectName>` to `projectName` so that folks can copy and paste the command directly without risk of missing deleting the `<>` which would unintendedly redirect input/output to a file in `cmd`